src/app/scenes/BookDetail.tsx

```diff
@@ -638,33 +644,12 @@ export class BookDetail extends React.Component<Props, State> {
                 </div>
               </section>
             ) : <BookDetailSectionPlaceholder />}
-            {seriesBookList &&
-              seriesBookList.length > 0 && (
-                <section className="PageBookDetail_Panel">
-                  <h2 className="PageBookDetail_PanelTitle">이 책의 시리즈</h2>
-                  <div className="PageBookDetail_PanelContent">
-                    <ConnectedInlineHorizontalBookList
-                      pageTitleForTracking="book-detail"
-                      uiPartTitleForTracking="series-list"
-                      disableInlineOnPC={seriesListExpanded}
-                      books={seriesBookList!}
-                      lazyloadThumbnail={false}
-                    />
-                    {!seriesListExpanded && seriesBookList.length > 6 &&
-                      !isMobile && (
-                        <div className="BookDetail_ContentTruncWrapper">
-                          <Expander
-                            onClick={() => {
-                              this.setState({ seriesListExpanded: true });
-                            }}
-                            text="펼쳐 보기"
-                            isExpanded={seriesListExpanded}
-                          />
-                        </div>
-                      )}
-                  </div>
-                </section>
-              )}
+            <CollapsableBookList
+              books={seriesBookList!}
```

Comment:
위 변경사항과 같이 적용할 수 있습니다.
```suggestion
              books={seriesBookList}
```